static/src/javascripts-legacy/bootstraps/enhanced/main.js

```diff
@@ -176,10 +176,13 @@ define([
             }, 'accessibility');
         }
 
-        if (config.page.nonKeywordTagIds && contains(config.page.nonKeywordTagIds.split(','), 'tone/recipes')) {
-            require(['bootstraps/enhanced/recipe-article'], function (recipes) {
-                bootstrapContext('recipes', recipes);
-            });
+        if (config.page.showNewRecipeDesign === true) {
+            //below is for during testing
+            if (config.tests.abNewRecipeDesign != undefined && config.tests.abNewRecipeDesign === true) {
+                require(['bootstraps/enhanced/recipe-article'], function (recipes) {
+                    bootstrapContext('recipes', recipes);
+                });
```

Comment:
Even if it is set to `false`, then the conditional will still be `false` so `if (config.page.showNewRecipeDesign)` is still accurate